src/engine/controllers/projectController.ts

```diff
@@ -119,6 +124,21 @@ export class ProjectController {
     };
   }
 
+  static checkProjectFileExist(context: Context, projectPath?: string): any {
+    context.logger.debug('load main yml file');
+    const config = ProjectController.loadConfigFile(context, projectPath);
+
+    context.logger.debug('load extensions');
+    const extensions = ProjectController.loadExtensions(config, projectPath);
+
+    const gqlSchema = GraphqlController.loadSchema(ProjectController.getSchemaPaths(extensions, projectPath));
+
+    return {
+      extensions: extensions,
+      gqlSchema: gqlSchema,
+    };
+  }
```

Comment:
it's strange that we return something from this function
```suggestion
    return { extensions, gqlSchema };
  }
```